lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -35,86 +35,189 @@ class NodeFinalize(Command):
     }
 
     def _validate_health_map(self):
+        check = True
         self.logger.debug("Validating node health check")
         resource_paths = [HEALTH_PATH]    #[HEALTH_PATH, MANIFEST_PATH]
         for resource in resource_paths:
             path = get_pillar_data(resource)
             if not path or path is MISSED:
-                raise CortxSetupError(f"{resource} resource is not configured")
+                check = False
+                self.logger.error(f"{resource} resource is not configured")
             path = path.split('://')[1]
             if not Path(path).is_file():
-                raise CortxSetupError(f"Validation failed: "
-                                      f"File not present {path}")
+                check = False
+                self.logger.error(f"Validation failed: File not present {path}")
         self.logger.debug("Node health check: Success")
+        return check
+
 
     def _validate_cert_installation(self):
+        check = True
         self.logger.debug("Validating certificate installtion check")
         cert_file = None
         cert_file = Path(CERT_PATH / 'stx.pem')
         if not cert_file.is_file():
-            raise CortxSetupError(
-                f"Validation failed: "
+            check = False
+            self.logger.error(
+                "Validation failed: "
                 f"Cert file not present {cert_file}"
             )
```

Comment:
After logging error lets return... Why are we printing error as well as success message

Reply:
Addressed